docs/Module-Review/Y4S1/CZ4124 Data Visualisation.md

```diff
@@ -0,0 +1,28 @@
+---
+id: cz4124-data-visualisation
+slug: cz4124-data-visualisation
+sidebar_position: 2
+title: CZ4124 Data Visualisation
+---
```

Comment:
Could you add in a short `description: ` field as well as it will appear on the card on the parent page.